addons/auth_signup/controllers/main.py

```diff
@@ -57,8 +57,8 @@ def web_auth_signup(self, *args, **kw):
                 if request.env["res.users"].sudo().search([("login", "=", qcontext.get("login"))]):
                     qcontext["error"] = _("Another user is already registered using this email address.")
                 else:
-                    _logger.error("%s", e)
-                    qcontext['error'] = _("Could not create a new account.")
+                    _logger.warning("%s", e)
+                    qcontext['error'] = _("Could not create a new account - %s", str(e))
```

Comment:
```suggestion
                    qcontext['error'] = _("Could not create a new account .") + "\n" + str(e))
```
so you don't have to modify the translation

Reply:
Hello @mart-e 
Changes have been done, Please check it.
Thanks :slightly_smiling_face: